src/systems/filecoin_mining/storage_mining/_index.md

```diff
@@ -19,36 +19,44 @@ The Filecoin Storage Mining Subsystem ensures a storage miner can effectively co
 The above involves a number of steps to putting on and maintaining online storage, such as:
 
 - Committing new storage (see Sealing and PoRep)
-- Continously proving storage ({see {<sref election_post>}})
+- Continously proving storage
 - Declaring storage faults and recovering from them.
 
-## Storage States
+## Sector Types
+
+There are two types of sectors, Regular Sectors with storage deals in them and Committed Capacity (CC) Sectors with no deals. All sectors require an expiration epoch that is declared upon PreCommit and sectors are assigned a StartEpoch at ProveCommit. Start and Expiration epoch collectively define the lifetime of a Sector. Length and size of active deals in a sector's lifetime determine the `DealWeight` of the sector. `SectorSize`, `Duration`, and `DealWeight` statically determine the power assigned to a sector that will remain constant throughout its lifetime. More details on cost and reward for different sector types will be announced soon.
+
```

Comment:
Explain why you would set up one sector type vs the other?

Reply:
it depends on the cost and reward structure for both sectors which will be announced soon.